src/server/pegasus_server_impl.cpp

```diff
@@ -2695,6 +2699,22 @@ void pegasus_server_impl::update_user_specified_compaction(
     }
 }
 
+void pegasus_server_impl::update_allow_ingest_behind(const std::map<std::string, std::string> &envs,
+                                                     bool default_value)
+{
+    const auto &iter = envs.find(ROCKSDB_ALLOW_INGEST_BEHIND);
+    if (iter == envs.end()) {
+        _allow_ingest_behind = false;
```

Comment:
I think you can set `_db_opts.allow_ingest_behind` directly

Reply:
Done